package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/pnpm/tarball-fetcher#readme",
   "dependencies": {
+    "@pnpm/logger": "^1.0.0",
```

Comment:
`@pnpm/logger` should be a peer dependency and installed as devDependency for testing.